### DISCUSSION TITLE
how to disable the scroller of entry?

### DISCUSSION THREAD

**Discussion by Francisundermoon** [fyne-io/fyne#3235] [Q&A] [answered]
`	input1 := widget.NewEntry()
	input1.MultiLine = true
	input1.Wrapping = fyne.TextTruncate
	input1.Resize(fyne.NewSize(180, 35))
	input1.Move(fyne.Position{
		X: 80,
		Y: 15,
	})`

I tried to run the code,and the entry shows vertical scroll and horizion scorll,even the entry is  filled with nothing.

could u   please tell me how to disable the entry scroller,tks alot!

**Comment by andydotxyz** [comment, marked as answer]
The `Resize` and `Move` would be run by your container, so should not impact this usage.
To disable the scroll functionality in `Entry` use `Wrapping = fyne.TextWrapOff`.
Beware that if you do this the content will force the UI to grow if the text gets longer.

**Reply by Francisundermoon** [reply]
tks buddy,I tried it,but the text content overflowed the entrybox.Is there a good way to hide the extra content?

**Reply by andydotxyz** [reply]
You cannot turn off scroll bars but keep scroll behaviour.

**Reply by Francisundermoon** [reply]
got it,tks again!
